package_build_list_host_darwin.json

```diff
@@ -25,7 +25,7 @@
         "OpenSSL-1.1.1o-rev1-mac": "package-system/OpenSSL/build_package_image.py",
         "OpenSSL-1.1.1o-rev1-ios": "package-system/OpenSSL/build_package_image.py --platform ios",
         "OpenEXR-3.1.3-rev4-mac": "Scripts/extras/pull_and_build_from_git.py ../../package-system/OpenEXR --platform-name Mac --package-root ../../package-system/OpenEXR/temp --clean",
-        "openimageio-opencolorio-2.3.12.0-rev2-mac": "package-system/openimageio-opencolorio/build_openimageio.py --clean",
+        "openimageio-opencolorio-2.3.17-rev2-mac": "package-system/openimageio-opencolorio/build_openimageio.py --package-name=openimageio-opencolorio-2.3.17-rev2-mac --clean",
```

Comment:
This works, although I added the `O3DE_PACKAGE_NAME` to avoid this duplication since it gets populated with the package name.